core/objects.go

```diff
@@ -269,6 +269,11 @@ type Challenge struct {
 	// Contains information about URLs used or redirected to and IPs resolved and
 	// used
 	ValidationRecord []ValidationRecord `json:"validationRecord,omitempty"`
+
+	// The account key used to create this challenge.  This is not part of the
+	// spec, but clients are required to ignore unknown fields, so it's harmless
+	// to include.
+	AccountKey *jose.JsonWebKey `json:"accountKey,omitempty"`
```

Comment:
Add a comment here explaining why we need to keep a copy of the account key on the challenge, and link to the issue.